diff --git a/dist/index.js b/dist/index.js
index 0ce9f9cc28070fef2005b88aef7c02d1ad08b032..c78be15e2dd16d9fcc4c94a84231856c427425d5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -4257,6 +4257,8 @@ function openTelemetryChannelSpanContext({
 }) {
   var _a24;
   if (!isNodeRuntime()) {
+    Promise.resolve(completion).catch(() => {
+    });
     return void 0;
   }
   const diagnosticsChannel = loadBuiltinModule(
